spec/features/auto_deploy_spec.rb

```diff
@@ -445,4 +448,54 @@
     expect(github).to_not receive(:create_deployment)
     push_event 'secret', head_commit: { id: "595ebd4ca061c4671ba89202aaf19c896f216635" }
   end
+
+  scenario 'Make sure DeploymentWatchdogWorker gets a job when deployment starts.' do
+    # make sure our queue is clear before starting test.
+    DeploymentWatchdogWorker.clear
+
+    # mock the repo's .slashdeploy.yml config.
+    config = <<-YAML.strip_heredoc
+    environments:
+      production:
+        continuous_delivery:
+          ref: refs/heads/master
+          required_contexts:
+            - ci/circleci
+    YAML
+
+    # I think this passes the YAML config into github as a contents symbol?
+    allow(github).to receive(:contents).and_return(config)
+
+    # prepare to receive a slack notification during push_event.
+    # these slack notification trigger when the config has required_context.
+    expect(slack).to receive(:direct_message).with \
+      slack_accounts(:david_baxterthehacker),
+      Slack::Message.new(text: ":wave: <@U012AB1AC>. I'll start a deployment of baxterthehacker/public-repo@0d1a26e to *production* for you once *ci/circleci* are passing.", attachments: [])
+
+    # our deployment watchdog worker should start with an empty queue.
+    assert_equal(0, DeploymentWatchdogWorker.jobs.size)
```

Comment:
The convention in codebases that use rspec is to use `expect()`.

```ruby
expect(DeploymentWatchdogWorker.jobs.size).to eq 0
```

Let's keep it consistent with the rest of the codebase.